### ISSUE TITLE
What problems could I have if I try to use Tapatalk with this add-on?

### ISSUE THREAD

**Issue by SpeedJack** [Xon/XenForo-WarningImprovements#1]
_Additional Requirements: **No Tapatalk**
It is unlikely that this addon will work as expected with Tapatalk_

But what **exactly** happens if I try to use Tapatalk and this add-on on my forum?
I need Tapatalk but it's not a problem for me if the only issue with using simultaneously these two add-ons is that moderators cannot warn from Tapatalk. While it's a problem if the application crashes or cause problems to users who receive warnings.
In short, I wish to know if there are problems only for moderators or also for users.

Can someone explain me exactly what kind of problems can cause the use of Tapatalk with this add-on?

_Thank you!_

**Comment by Xon** [comment]
I simply haven't tested Tapatalk, and am unsure about how my add-ons will work with it.

I suspect it should work as-is, as the only controller (aka front-end) changes are display the Warning Actions on a given account.
